packages/www/components/Dashboard/WebhooksTable/index.tsx

```diff
@@ -118,6 +129,36 @@ const WebhooksTable = ({ title = "Webhooks" }: { title?: string }) => {
                 cursor: "pointer",
               },
             },
+            status: {
+              children: (
+                <Box>
+                  {!webhook.status ? (
+                    <StatusBadge
+                      variant={StatusVariant.Idle}
+                      tooltipText="No triggers yet"
+                    />
+                  ) : webhook.status.lastFailure &&
+                    +Date.now() - webhook.status.lastFailure.timestamp <
+                      WARNING_TIMEFRAME ? (
+                    <StatusBadge
+                      variant={StatusVariant.Unhealthy}
+                      timestamp={webhook.status.lastFailure.timestamp}
+                      tooltipText="Last failure"
+                    />
+                  ) : (
+                    <StatusBadge
+                      variant={StatusVariant.Healthy}
+                      timestamp={webhook.status.lastTriggeredAt}
+                      tooltipText="Last triggered"
+                    />
+                  )}
+                </Box>
```

Comment:
Neat!

Reply:
Spotted a issue! If `lastTriggeredAt` is higher than `WARNING_TIMEFRAME` and is failed, it becomes `Healthy`! Fixed now